scenarios/chained-cpu-congestion/plan.yaml

```diff
@@ -0,0 +1,82 @@
+---
+phases:
+  - name: Phase1
+    repeat: 1
+    client:
+      workers:
+        - instances: 5
+          duration: 10m
+          delay: 2s
+
+    workload:
+      actions:
+        - name: AllocateMemory
+          config:
+            sizeBytes: 1000
+            numAllocations: 1000
+        
+        - name: Burn
+          config:
+            duration: 20ms
+
+        # Send request to payment
+        - name: HTTPRequest
+          config:
+            url: http://payment:8080
+            body:
+              actions:
+                - name: AllocateMemory
+                  config:
+                    sizeBytes: 1000
+                    numAllocations: 1000
+                
+                - name: Burn
+                  config:
+                    duration: 10ms
+        # Stress Inventory Service and propagate to Order Service
+                - name: HTTPRequest
+                  config:
+                    url: http://order-service:8080
+                    body:
+                      actions:
+                        - name: Burn
+                          config:
+                            duration: 1000ms
+
+                        # Sleep to simulate delay before Payment Service is impacted
+                        - name: Sleep
+                          config:
+                            duration: 10s
+
+                        - name: HTTPRequest
+                          config:
+                            url: http://payment-service:8080
+                            body:
+                              actions:
+                                - name: AllocateMemory
+                                  config:
+                                    sizeBytes: 200
+                                    numAllocations: 200
+
+                                # Sleep to simulate delay before Frontend Service is impacted
+                                - name: Sleep
+                                  config:
+                                    duration: 10s
+
+                                - name: HTTPRequest
+                                  config:
+                                    url: http://frontend-service:8080
```

Comment:
Looks like payment is invoking front end, instead of front end invoking payment

Reply:
fixed